scripts/vaccination_alerts/parse-vaccination-phases.ts

```diff
@@ -63,6 +63,7 @@ const buildRegionGroup = (
     locationName: state,
     fips: regions.findByFullName(state.trimRight())!.fipsCode,
     eligibilityInfoUrl: firstRow.eligibilityUrl,
+    vaccinationSignupUrl: '', // this does not appear to be available in the spreadsheet
```

Comment:
FWIW- Brett says we can delete this script, so feel free to do that if it's easier.

Reply:
nice.